install_config/topics/escaping_special_characters.adoc

```diff
@@ -0,0 +1,17 @@
+////
+Escaping special characters in ansible variables file
+
+This module included in the following assemblies:
+* install_config/configuring_openstack.adoc
+* install_config/install/advanced_install.adoc
+////
+
+[IMPORTANT]
+====
+If you are using special characters (for example, `pass:[#]`, `{` or `}`) in the
```

Comment:
I might replace the first sentence: If a parameter value in the Ansible inventory file contains special characters, such as `#`, `{` or `}`, you must enclose the value in both single and double quotation marks.